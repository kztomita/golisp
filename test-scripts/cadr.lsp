(print (cadr '(1 2 3 4)))
(print (cadr '()))
(print (cadr nil))

(print (cddr '(1 2 3 4)))
(print (cddr '()))
(print (cddr nil))
