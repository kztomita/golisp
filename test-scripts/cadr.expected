2
nil
nil
(3 4)
nil
nil
